test/ProgressOnderwijsUtils.Tests/Data/DatabaseDescriptionTest.cs

```diff
@@ -36,5 +36,29 @@ public void AllDependantTables_works_recursively()
             PAssert.That(() => SomeDataWithDefault_metadata.HasDefaultValue);
             PAssert.That(() => !SomeDataWithoutDefault_metadata.HasDefaultValue);
         }
+
+        [Fact]
+        public void CheckConstraintWithTable_works()
+        {
+            SQL($@"
+                create table dbo.CheckConstraintTest (
+                    IdRoot int not null primary key,
+                    Test int not null
+                );
+
+                alter table dbo.CheckConstraintTest add constraint ck_TestConstraint check (Test <> 0);

```

Comment:
Maakt verder niet uit, maar je kan deze constraint ook gelijk in de create table definiëren. Vind ik net wat leesbaarder. Maar zo ook goed natuurlijk.